docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,137 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with/without storage providers
+
+## Prerequisites 
+
+
+### [Optional] Installing [k9s](https://k9scli.io/) 
```

Comment:
Please add the link to the k9s installation page https://k9scli.io/topics/install/

Also, according to the k9s installation page, the command for mac/linus is `brew install derailed/k9s/k9s`. Please check if it's interchangeable or not